tests/week-2/c-comparison-operators.test.js

```diff
@@ -0,0 +1,34 @@
+const { importSourceAndRead } = require("../helpers");
+require("../../week-2/C-comparison-operators/exercise.js");
+
+let contents = "";
+
+importSourceAndRead(
+  require,
+  "../../week-2/C-comparison-operators/exercise.js"
+).then(res => (contents = res));
+
+test("Comparison statement is written correctly for 'moreStudentsThanMentors'", () => {
+  const hasCorrectKeywords = !!contents.match(
+    /moreStudentsThanMentors\s*=\s*studentCount\s*>\s*mentorCount/
+  );
+  expect(hasCorrectKeywords).toBe(true);
+});
+
+test("Comparison statement is written correctly for 'enoughSpaceInRoom'", () => {
+  const hasCorrectKeywords =
+    !!contents.match(
+      /enoughSpaceInRoom\s*=\s*roomMaxCapacity\s*>=?\s*studentCount\s*\+\s*mentorCount/
+    ) ||
+    !!contents.match(
+      /enoughSpaceInRoom\s*=\s*roomMaxCapacity\s*>=?\s*mentorCount\s*\+\s*studentCount/
+    );
+  expect(hasCorrectKeywords).toBe(true);
```

Comment:
could we please add two more checks to the above test so below passes:

`enoughSpaceInRoom = mentorCount + studentCount <= roomMaxCapacity`
`enoughSpaceInRoom = studentCount + mentorCount <= roomMaxCapacity`

Reply:
Did not think of that! Fixed !